host/libhackrf/src/hackrf.c

```diff
@@ -1310,9 +1250,7 @@ int ADDCALL hackrf_version_string_read(hackrf_device* device, char* version,
 	}
 }
 
-extern ADDAPI int ADDCALL hackrf_usb_api_version_read(hackrf_device* device,
-		uint16_t* version)
-{
+extern ADDAPI int ADDCALL hackrf_usb_api_version_read(hackrf_device* device, uint16_t* version) {
```

Comment:
Sometimes the automatic formatting goes against the manual efforts to increase readability. Which makes me wonder if it's really a good idea to hand over the intelligent efforts of the programmer to a rigid algorithm. It does well in many cases, and surely has some value, but at what cost? (Just a general point, because I really appreciate your efforts and see that a lot of the changes are increasing readability and understanding of the code. And I'm only criticizing the automatic reformatting. Maybe the settings could be tweaked further?)

Reply:
I don't see any manual effort to improve readability here, just a line that was previously wrapped in 80 characters which has now been left on a single line because we chose a line width of 100 characters.

We could choose a shorter line width - I started with 100 because it seemed to largely reflect existing practice.

One argument for a shorter length would be that GitHub ends up displaying a horizontal scrollbar for many of the diffs you've commented on...

Reply:
I've reduced the line width to 90 characters, so this function header is now split over multiple lines again.